sdk/core/core-client/src/interfaces.ts

```diff
@@ -374,28 +408,73 @@ export interface Serializer {
   ): any;
 }
 
+/**
+ * Constraints used to test values.
```

Comment:
I think the word "test" could be confusing in this context. Perhaps this could work:
```suggestion
 * Description of various value constraints such as integer ranges and string regex.
```

Reply:
Thank you! I will update this PR tomorrow. Please send me all the feedback 🌞